README.md

```diff
@@ -69,6 +69,15 @@ $ cd lorri
 $ nix-env -if .
 ```
 
+### (for non NixOS) Create gcroots directory
+
+lorri uses the `/nix/var/nix/gcroots/per-user/$USER` directory which
+is present in NixOS installation, but not always otherwise:
+
+```
+$ mkdir -m 0755 -p /nix/var/nix/gcroots/per-user/$USER
```

Comment:
This is fixed by https://github.com/target/lorri/pull/28 so we don’t need this bit of documentation anymore.